### ISSUE TITLE
Indentation: Undetected spaces when using tabs for indentation

### ISSUE THREAD

**Issue by ntwb** [stylelint/stylelint#389]
Given the following CSS:

```css
/**
 * HiDPI Displays
 */
@media print,
  (-webkit-min-device-pixel-ratio: 1.25),
  (min-resolution: 120dpi) {

	.spinner {
		background-image: url(../images/spinner-2x.gif);
	}

}
```

Lines 5 and 6 above are using 2 spaces for indentation and these spaces are not detected and no warning is given when using this rule:
```
"indentation": [ 2, "tab" ],
```

Note: Lines 8, 9, 10 do use tabs for indentation

**Comment by davidtheclark** [comment]
Now by default the indentation rule checks the indentation that you want it to check (but you can turn this off with `except: ["param"]`).

**Comment by ntwb** [comment]
Confirming as fixed :+1:
